shard-operator/src/main/java/com/redhat/service/smartevents/shard/operator/ManagerSyncServiceImpl.java

```diff
@@ -62,6 +62,7 @@ protected Uni<Object> doBridges() {
         return managerClient.fetchBridgesToDeployOrDelete()
                 .onItem().transformToUni(x -> Uni.createFrom().item(x.stream().map(y -> {
                     if (y.getStatus().equals(ManagedResourceStatus.PREPARING)) { // Bridges to deploy
```

Comment:
what is we add here `if (y.getStatus().equals(ManagedResourceStatus.PREPARING) || y.getStatus().equals(ManagedResourceStatus.PROVISIONING))`? 

I know we would send another `PROVISIONING` update to the manager but I think it would be anyway more consistent

Reply:
To be honest I prefer the separation.

We won't get the duplicate (misleading) status update and the log entries are more contextually correct.

Let's wait for my revised PR to go to review and see what everybody thinks.